src/collectd.conf.pod

```diff
@@ -5563,9 +5563,9 @@ Enable the collection of primary / replica statistics in a replication setup. In
 order to be able to get access to these statistics, the user needs special
 privileges. See the B<User> documentation above. Defaults to B<false>.
 
-=item B<MariaDb> I<true|false>
+=item B<MariadbDialect> I<true|false>
 
-Enable Multisource replication Mariadb support.
+Enable Multisource replication MariadbDialect support.
```

Comment:
This may sound like splitting hairs or overly critical, but what should "MariadbDialect" be? I think this needs a little bit more explanation, what exactly does this switch do?

Reply:
ok, added like in telegraf :)